index.js

```diff
@@ -17,6 +17,7 @@ async function onTrack(event, settings) {
             api_key: settings.apiKey,
             properties: {
                 ...event.properties,
+                $browser: determineBrowser(navigator.userAgent, navigator.vendor, window.opera),
```

Comment:
Couldn't we just do `...(event.properties.browser ? {$browser: event.properties.browser} : {})`

Reply:
I'm unfamiliar with how Segment works. Will that browser property already exist at this point? This would indeed be the ideal solution

Reply:
Yep it should do